spec/models/video_spec.rb

```diff
@@ -4,4 +4,39 @@
   it { should belong_to(:category) }
   it { should validate_presence_of(:title) }
   it { should validate_presence_of(:description) }
+
+  describe "#search_by_title" do
```

Comment:
Instead of `#` It should be `.` - RSpec has some 'rules' when it comes to describe messages. Read more here -> http://betterspecs.org/#describe